src/pyamsd/commands/to_csv.py

```diff
@@ -14,8 +14,17 @@
 from pyamsd.util import *  # noqa: F403
 
 
+def register(parser):
+    parser.add_argument(
+        'check',
+        action='store', default='', nargs='?', choices=['check', '']
+    )
```

Comment:
Looks like a boolean flag to me, right? So maybe rather
```python
parser.add_argument('--check', action='store_true', default=False, help='Only check the data')
```

For `git` (and other cli programs I've seen), an option to "try out" an otherwise state changing command is called `--dry-run`. Maybe that would be a good name here, too?

Reply:
`--dry-run` makes it clearer, `check` was only taken due to history